### DISCUSSION TITLE
Is this a dead project?

### DISCUSSION THREAD

**Discussion by Sunspark-007** [89luca89/distrobox#1581] [General]
Last release 5 months ago.
Many issues posted, some are showstoppers/blockers.
Pull requests exist.
89luca89 hasn't participated at any public level for distrobox since August.

**Comment by mirkobrombin** [comment]
Definitely not, we are monitoring bugs and discuss potential fixes. We are also focusing on improving the documentation and the overall identity.

**Comment by Sunspark-007** [comment]
One reason I wondered is because distrobox used to work on a Steam Deck with 3.4 as a user install using the original install scripts and documentation but now no longer does with the current install scripts and documentation, which means userspace was broken. The OS didn't change, but the install scripts did. I did pretty good identifying that podman is completely busted but lilipod still mostly worked, but the rest of it I can't get going because the script has changed too much.

I submitted issues on it 2 months ago, the main one being below, and while I didn't expect instant turnaround I figured it would have been tagged by someone as something to look into as a mostly easy fix because it likely affects other systems that are immutable-based and was something that used to work and not new functionality required.

One could say well, just use whatever was bundled if it works, but the documentation says one of the benefits of having a user-setup version is that you can make updates instead of waiting for a frozen distro to update.

https://github.com/89luca89/distrobox/issues/1510

**Comment by 89luca89** [comment]
@Sunspark-007 it's not a dead project I can confirm

Inactivity periods are because real life is a thing, and I don't get paid to work on distrobox (nor I want to)
So either check on the matrix channel or in discussion if someone else can help, or you investigate it
Or wait, your choice

There is no SLAs, no ETAs and no timed support to expect
